### PR TITLE
docs: updates relative slowness of KMS v1 compared to KMS v2

### PR DESCRIPTION
**Pull request by nilekhc**

### CODE REVIEW

**Comment by nilekhc** [comment]
@ritazh @enj @aramase Let me know if I am missing anything.

File: content/en/docs/tasks/administer-cluster/encrypt-data.md [hunk 1]
```diff
@@ -101,7 +101,8 @@ Name | Encryption | Strength | Speed | Key Length | Other Considerations
...
+`kms v1` | Uses envelope encryption scheme: Data is encrypted by data encryption keys (DEKs) using AES-CBC with [PKCS#7](https://datatracker.ietf.org/doc/html/rfc2315) padding (prior to v1.25), using AES-GCM starting from v1.25, DEKs are encrypted by key encryption keys (KEKs) according to configuration in Key Management Service (KMS) | Strongest | Slow(_compared to `kms v2`_) | 32-bytes |  The recommended choice for using a third party tool for key management. Simplifies key rotation, with a new DEK generated for each encryption, and KEK rotation controlled by the user. [Configure the KMS V1 provider](/docs/tasks/administer-cluster/kms-provider#configuring-the-kms-provider-kms-v1).
```
Comment (ritazh) [inline content/en/docs/tasks/administer-cluster/encrypt-data.md]: Should we recommend kms v2 over kms v1?

File: content/en/docs/tasks/administer-cluster/encrypt-data.md [hunk 2]
```diff
@@ -101,7 +101,8 @@ Name | Encryption | Strength | Speed | Key Length | Other Considerations
...
+`kms v2` | Uses envelope encryption scheme: Data is encrypted by data encryption keys (DEKs) using AES-GCM, DEKs are encrypted by key encryption keys (KEKs) according to configuration in Key Management Service (KMS) | Strongest | Fast | 32-bytes |  The recommended choice for using a third party tool for key management. A new DEK is generated at startup and reused for encryption. The DEK is rotated when the KEK is rotated. [Configure the KMS V2 provider](/docs/tasks/administer-cluster/kms-provider#configuring-the-kms-provider-kms-v2).
```
Comment (ritazh) [inline content/en/docs/tasks/administer-cluster/encrypt-data.md]: Should we say beta after 1.27?

File: content/en/docs/tasks/administer-cluster/encrypt-data.md (hunk 1 above)
Comment (nilekhc) [reply content/en/docs/tasks/administer-cluster/encrypt-data.md]: Should we remove v1 altogether? I thought this table lists all available providers.

Comment (aramase) [reply content/en/docs/tasks/administer-cluster/encrypt-data.md]: I think `The recommended choice for using a third party tool for key management.` should only be for kms v2 but we should keep v1 in the list until it's supported.

**Comment by ritazh** [comment]
LGTM

File: content/en/docs/tasks/administer-cluster/encrypt-data.md [hunk 3]
```diff
@@ -101,7 +101,8 @@ Name | Encryption | Strength | Speed | Key Length | Other Considerations
...
+`kms v1` | Uses envelope encryption scheme: Data is encrypted by data encryption keys (DEKs) using AES-CBC with [PKCS#7](https://datatracker.ietf.org/doc/html/rfc2315) padding (prior to v1.25), using AES-GCM starting from v1.25, DEKs are encrypted by key encryption keys (KEKs) according to configuration in Key Management Service (KMS) | Strongest | Slow (_compared to `kms v2`_) | 32-bytes |  Simplifies key rotation, with a new DEK generated for each encryption, and KEK rotation controlled by the user. [Configure the KMS V1 provider](/docs/tasks/administer-cluster/kms-provider#configuring-the-kms-provider-kms-v1).
+`kms v2` | Uses envelope encryption scheme: Data is encrypted by data encryption keys (DEKs) using AES-GCM, DEKs are encrypted by key encryption keys (KEKs) according to configuration in Key Management Service (KMS) | Strongest | Fast | 32-bytes |  The recommended choice for using a third party tool for key management. Available in beta after `1.27`. A new DEK is generated at startup and reused for encryption. The DEK is rotated when the KEK is rotated. [Configure the KMS V2 provider](/docs/tasks/administer-cluster/kms-provider#configuring-the-kms-provider-kms-v2).
```
Comment (aramase) [inline content/en/docs/tasks/administer-cluster/encrypt-data.md]:
```suggestion
`kms v2` | Uses envelope encryption scheme: Data is encrypted by data encryption keys (DEKs) using AES-GCM, DEKs are encrypted by key encryption keys (KEKs) according to configuration in Key Management Service (KMS) | Strongest | Fast | 32-bytes |  The recommended choice for using a third party tool for key management. Available in beta from `v1.27`. A new DEK is generated at startup and reused for encryption. The DEK is rotated when the KEK is rotated. [Configure the KMS V2 provider](/docs/tasks/administer-cluster/kms-provider#configuring-the-kms-provider-kms-v2).
```

**Review by aramase — CHANGES_REQUESTED** [review]

**Review by aramase — APPROVED** [review]

**Comment by enj** [comment]
LGTM

**Review by ritazh — APPROVED** [review]

**Pull request merged**